SwiftWisdomTests/StandardLibrary/Numbers/UnsignedInteger+ExtensionsTests.swift

```diff
@@ -19,4 +19,134 @@ class UnsignedIntegerExtensionTests: XCTestCase {
         let mixed = -100...100
         XCTAssert(mixed.contains(random(inRange: mixed)))
     }
+    
+    func test_ip_initData() {
+        var data = Data(byte: UInt8(0))
+        var number = UInt8.init(ip_data: data)
```

Comment:
let's remove .init from this test and the rest of the tests in this diff - `UInt8(...)` should do.